### DISCUSSION TITLE
We do not allow applicants to apply using this email domain. Please select a different email address.

### DISCUSSION THREAD

**Discussion by khem18pun199805** [community/community#134912] [GitHub Education]
### Hi everyone,

why my account expire and ask for reverification but when i go to reverification page and got this type msg "We do not allow applicants to apply using this email domain. Please select a different email address."

**Comment by srijon57** [comment]
#Check Email Domain: Ensure that you are using an official email address provided by your educational institution (e.g., @university.edu).

#GitHub Support: If you believe your email should be accepted, reach out to  (https://support.github.com/contact) for further assistance. Provide them with details about your situation and any supporting documents that verify your enrollment.

#Alternative Email: If possible, use an alternative email address that meets GitHub Education’s criteria.

**Reply by khem18pun199805** [reply]
yes i am using officially email and in this world useless support system is the only on site is GitHub support. so i am here to get soln.
